### PR TITLE
first commit- variables, who we are, global wrapper, styles

### PR DESCRIPTION
**Pull request by liannarice**
Also includes also borbox, and basics of media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,12 @@
     <title>If These Lands Could Talk</title>
     <link rel="stylesheet" href="./css/normalize.css" />
     <link rel="stylesheet" href="./css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
Comment (SolivanLau) [inline index.html:11]: Looks great - there looks to be a weird formatting error (maybe via prettier) we can  fix that in a later PR request

**Review by SolivanLau — APPROVED** [review]
Looks great! thank you for following the comment outline for adding css. It follows the cascade and code overall is approved by me and ready to me merged into the main branch

File: css/style.css [hunk 2]
```diff
@@ -1,13 +1,137 @@
 /* BOX SIZING */
 
+html { box-sizing: border-box; }
+
+*, *:before, *:after { box-sizing: inherit; }
+
+
 /* VARIABLES */
```
Comment (maxharrisnet) [inline css/style.css:9]: Here's an example of a comment

File: css/style.css [hunk 3]
```diff
@@ -1,13 +1,137 @@
...
+    --light-teal: #00c6d0;
+    --cybergrape: #6C4681;
+    --onyx: #424241;
+    --yaleblue: #00478F;
+    --hotpink: #FA6CB7;
+    --citrine: #D3C53F;
+    --citron: #92A32D;
+    --bcitron: #B37122;
+
+    /* font sizes */
```
Comment (maxharrisnet) [inline css/style.css:52]: This looks good, great use of rems!

**Pull request merged**